app/assets/stylesheets/custom.scss

```diff
@@ -213,6 +210,7 @@ header {
     a {
       font-weight: normal;
       font-size: 1rem;
+      text-decoration: none;
```

Comment:
What's the reason for this change? :thinking:

Reply:
I will remove this change